### ISSUE TITLE
why i scan a barcode there is no result return to me ?

### ISSUE THREAD

**Issue by kibyOu** [code-kotis/barcode-scanner#20]

**Comment by gokulkrishh** [comment]
@kibyOu You could be more specific. Can you post the barcode which you scanned ??

**Comment by gokulkrishh** [comment]
@kibyOu Hey can you give some more info on this. Like which QR code you scanned and what was the issue?

**Comment by anundinesh** [comment]
I have used "code 39" barcodes, it is not scanning them.

**Comment by gokulkrishh** [comment]
@anundinesh Can you post some samples ??

**Comment by anundinesh** [comment]
go to this https://racoindustries.com/barcodegenerator/1d/code-39/
generate a barcode with this "AT09A00001" and check.

**Comment by gokulkrishh** [comment]
@anundinesh Yes its not working. I have tried couple of other barcodes. None of them worked.

**Comment by anundinesh** [comment]
@gokulkrishh do you have any solution for it?

**Comment by gokulkrishh** [comment]
@anundinesh Only solution is to change the decoder I use for decoding image to get code. I will look into it.
